route/include/route_generator_worker.h

```diff
@@ -190,6 +183,13 @@ namespace route {
         */
         void set_CTE_dist(double cte_dist);
 
+        /**
+         * \brief Get the closest lanelet from the list of llts
```

Comment:
"Get the closest lanelet on the route from the list of <distance, lanelet> pairs relative to the vehicle's current point. If the input list does not contain lanelets on the route, still closest lanelet from the route will be returned"